todo_app/trello_api.py

```diff
@@ -0,0 +1,82 @@
+import os
+import requests
+
+class TrelloApi:
+
+  base_url = os.getenv('TRELLO_API_BASE_URL')
+  board_id = os.getenv('TRELLO_BOARD_ID')
+  token = os.getenv('TRELLO_API_TOKEN')
+  key = os.getenv('TRELLO_API_KEY')
```

Comment:
This works nicely here, but pre-loading these may bite you in the next module so worth keeping an eye out for it.

The reason it could be a problem is that these class variables are initialised when this file is first _imported_. That works perfectly right now, but:
* When we introduce tests we'll sometimes want to do some setup before we start the app
* This can include loading fake/test environment variables
* But these variables will already have had their values assigned by the time we get to our setup, so we won't affect them

We can fix that by messing with our import order carefully, but we typically end up with very fragile code, so a better solution is one of:
* The easiest here is probably to load the environment variables from functions when they're needed, e.g. in `get_cards`:
```python
url = f"{cls.get_base_url()}/1/boards/{cls.get_board_id()}/lists"
```
or even just loading the environment variables directly:
```python
url = f"{os.getenv('TRELLO_API_BASE_URL')}/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"
```
* Personally I'd be lazy and go with the above here, but alternatively we could avoid using them statically, and load the environment variables in a constructor, e.g. something like:
```python
class TrelloApi():

    def __init__(self):
        self.base_url= os.getenv('TRELLO_API_BASE_URL')
        ...

    def get_cards(self):
        url = f"{self.base_url}/1/boards/{self.board_id}/lists"
        ...
```

Reply:
That makes sense, thanks! I've opted to change the class so that the methods are no longer class methods and all the variables are accessed via self.